npdarrington/fitlit
# Nd display dashboard

What’s this PR do?  
- Rebuilds the new  UI and displays the information needed by a user's hydration, activity, user profile, leaderboards and sleep.
- Includes chart.js to display a graph for showing a user's hydration over a week.
- Large DOM rework.
- Large scripts.js rework.
- Small modifications to CSS and HTML from previous branch that was more of a shell of the new UI to have the data display correctly.
- Started cleaning up unused variable declarations.
 
Where should the reviewer start?  
- Git pull latest updates.
 
How should this be manually tested?  
- npm install to install dependencies for chart.js
- npm test to ensure all tests are still passing (some classes had updates to display latest date in dataset).
- open index.html from terminal.
- Click between the different icons on the bottom left to shift between the data.
 
Any background context you want to provide?  
- This PR does not include displaying the challenge between friends in iteration 5
- This PR does not include: Activity - How their number of steps, minutes active, and flights of stairs climbed compares to all users for the latest day from rubric.
 
What are the relevant tickets?  
Closes #55 
 
Screenshots (if appropriate)  
- N/A
 
Questions: 
- N/A
